Procfile.dev

```diff
@@ -0,0 +1,2 @@
+web: rails s -p 3000
+client: bin/webpack-dev-server
```

Comment:
Can we change `client:` to `webpack:` please? it will make it clearer what process is running in the log output imo.

Reply:
> Hey @arku, thanks for doing this. We don't have foreman in the Gemfile currently so we will need to add that with this as well I think.

https://github.com/ddollar/foreman/wiki/Don't-Bundle-Foreman

> If you create a .foreman file in the root of the directory and add this to it: procfile: Procfile.dev we can start the server with just foreman start without the flags and make it a bit simpler. What do you think?

Good idea! I'm on it.

Reply:
Done.